resources/js/src/app/components/customer/login/GuestLogin.js

```diff
@@ -25,51 +43,45 @@ Vue.component("guest-login", {
     {
         this.$nextTick(() =>
         {
+            // for old login view only (input in modal)
             $("#guestLogin").on("hidden.bs.modal", () =>
```

Comment:
Wir sollten das Element nicht über die Id referenzieren. Bitte `this.$refs` verwenden

Reply:
done
https://github.com/plentymarkets/plugin-ceres/pull/1413/commits/4b09e05228e36263d32b555c6b0ddb5d986709f9